ntbs-service/Pages/ServiceDirectory/Index.cshtml

```diff
@@ -0,0 +1,25 @@
+﻿@page
+@model ntbs_service.Pages.ServiceDirectory.IndexModel
+
+@{
+    Layout = "Shared/_Layout";
+}
+
+<nhs-width-container container-width="Standard">
+    <h2 class="nhsuk-heading-l">Service directory</h2>
+    <h3 class="nhsuk-heading-m"> Regions </h3>
+    
+    <div>
+        @foreach (var regionGroupOfFour in Model.AllRegionsGrouped)
+        {
+            <nhs-grid-row>
+                @foreach (var region in regionGroupOfFour)
+                {
+                    <nhs-grid-column grid-column-width="OneQuarter">
+                        <a class="nhsuk-heading-m" href=@($"/ServiceDirectory/Region/{region.Code}")>@region.Name</a>
+                    </nhs-grid-column>
+                }
+            </nhs-grid-row>
+        }
```

Comment:
you don't need to do this whole splitting up into fours by hand.
If you just spit out 1/4-sized columns, then they get shifted to next rows automatically as needed, e.g.:
```
<nhs-grid-column grid-column-width="OneQuarter">A</nhs-grid-column>
<nhs-grid-column grid-column-width="OneQuarter">B</nhs-grid-column>
<nhs-grid-column grid-column-width="OneQuarter">C</nhs-grid-column>
<nhs-grid-column grid-column-width="OneQuarter">D</nhs-grid-column>
// This will be on the next line automatically!!
<nhs-grid-column grid-column-width="OneQuarter">E</nhs-grid-column>
```

I'm not actually sure whether the design framework uses a css `grid` or css `flexbox` to get this behaviour (I think the latter), but the gist is, we can rely on it here!